e2e/benchmarks/browserstack-benchmark/app.js

```diff
@@ -362,6 +363,12 @@ function setupHelpMessage() {
     help: 'runs GCP compatible version of benchmarking system',
     action: 'store_true'
   });
+  parser.add_argument('--weeklyRun', {
```

Comment:
Nit: `--weeklyRun` implies it runs once every week.

```suggestion
  parser.add_argument('--dailyRun', {
```

Reply:
Good suggestion. What about 'weeklyCycleRun'? In addition to distinguishing from 'weekly run', it indicates 'weekly cycle'?

Reply:
Sounds good to me.

Reply:
Thank you!